src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+class Truck extends Machine {
```

Comment:
The class `Truck` should be declared as public to match the visibility of its superclass `Machine` and to be accessible from other packages.